frappe/public/js/frappe/desk.js

```diff
@@ -281,13 +281,7 @@ frappe.Application = class Application {
 				frappe.dom.set_style(frappe.boot.print_css, "print-style");
 			}
 
-			let current_app = localStorage.current_app;
-			if (current_app) {
-				frappe.boot.setup_complete =
-					frappe.boot.setup_wizard_not_required_apps?.includes(current_app) ||
-					frappe.boot.setup_wizard_completed_apps?.includes(current_app);
```

Comment:
This seems to be the only place (in the frontend) where these values are used, so we can likely remove them from `boot.py` as well.

Reply:
We do have this function `remove_apps_with_incomplete_dependencies`, but I'm not sure if this function's working has some issues or not. Or should I remove `setup_wizard_not_required_apps` and `setup_wizard_completed_apps` with this function only? 🤔

Reply:
Ref PR: https://github.com/frappe/frappe/pull/32640

Reply:
@rohitwaghchaure why did you mark this as resolved? `setup_wizard_not_required_apps` will have no frontend uses after this PR.

Reply:
The `setup_wizard_completed_apps`, yes. But not the `setup_wizard_not_required_apps`.

Reply:
#33546